### PR TITLE
Fix:  Update overview stage

### PR DESCRIPTION
**Pull request by mykalmax**
simplifies Plan Interface a bit 

https://github.com/TobikoData/sqlmesh/assets/5644753/8f3ddfa4-90a8-4355-9ca6-1f350b1a0491

### CODE REVIEW

**Comment by tobymao** [comment]
we shouln't display fqn, we should always display model names or display names. it should match the cli. so you should never see memory.x.y

**Comment by mykalmax** [comment]
> we shouln't display fqn, we should always display model names or display names. it should match the cli. so you should never see memory.x.y

`fqn` fixes were in another PR and video clip is made before those changes merged.

File: web/client/src/library/components/plan/PlanActionsDescription.tsx [hunk 1]
```diff
@@ -1,77 +1,97 @@
...
+            </span>
+          )}
+          {isTrue(no_auto_categorization) && (
+            <span className="inline-block mr-1">
+              also set <b>Change Category</b> manually
+            </span>
+          )}
+          {isFalse(isStringEmptyOrNil(restate_models)) && (
+            <span className="inline-block mr-1">
+              and restate the followingmodels <b>{restate_models}</b>
```
Comment (vchan) [inline web/client/src/library/components/plan/PlanActionsDescription.tsx:87]:
```suggestion
              and restate the following models <b>{restate_models}</b>
```

File: web/client/src/library/components/plan/PlanActionsDescription.tsx [hunk 2]
```diff
@@ -1,77 +1,97 @@
...
+          {isTrue(skip_backfill) && (
+            <span className="inline-block mr-1">
+              without <b>Backfills</b>
+            </span>
+          )}
+          {isTrue(forward_only) && (
+            <span className="inline-block mr-1">
+              consider as a <b>Breaking Change</b>
+            </span>
+          )}
```
Comment (vchan) [inline web/client/src/library/components/plan/PlanActionsDescription.tsx:79]: Is this right?

**Review by vchan — APPROVED** [review]

File: web/client/src/library/components/plan/PlanActionsDescription.tsx (hunk 2 above)
Comment (mykalmax) [reply web/client/src/library/components/plan/PlanActionsDescription.tsx]: changed text to Forward Only

**Pull request merged**